SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/haoze/flaky-reproduction/systems/hdfs-4233/hadoop-dist/target/hadoop-0.23.6-SNAPSHOT/share/hadoop/common/lib/slf4j-log4j12-1.6.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/haoze/tmp/bytecode/hdfs-4233/runtime-1.0-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
Exception in thread "main" java.io.IOException: failure to login
	at org.apache.hadoop.security.UserGroupInformation.getLoginUser(UserGroupInformation.java:521)
	at org.apache.hadoop.security.UserGroupInformation.getCurrentUser(UserGroupInformation.java:480)
	at org.apache.hadoop.ipc.RPC.waitForProtocolProxy(RPC.java:279)
	at org.apache.hadoop.ipc.RPC.waitForProtocolProxy(RPC.java:232)
	at org.apache.hadoop.ipc.RPC.waitForProtocolProxy(RPC.java:194)
	at org.apache.hadoop.ipc.RPC.waitForProxy(RPC.java:176)
	at org.apache.hadoop.hdfs.server.namenode.SecondaryNameNode.initialize(SecondaryNameNode.java:221)
	at org.apache.hadoop.hdfs.server.namenode.SecondaryNameNode.<init>(SecondaryNameNode.java:180)
	at org.apache.hadoop.hdfs.server.namenode.SecondaryNameNode.main(SecondaryNameNode.java:645)
Caused by: javax.security.auth.login.LoginException: flaky test exception injection of TraceAgent
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at runtime.TraceAgent.createException(TraceAgent.java:128)
	at runtime.TraceAgent.<clinit>(TraceAgent.java:151)
	at org.apache.hadoop.hdfs.server.namenode.SecondaryNameNode.main(SecondaryNameNode.java)
